angular.json

```diff
@@ -135,12 +136,16 @@
       "prefix": "app",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-devkit/build-angular:application",
           "options": {
-            "outputPath": "dist/quickstart-demo",
+            "outputPath": {
+              "base": "dist/quickstart-demo",
+              "browser": ""
```

Comment:
Adding `"browser": ""` to a few places was one of the few manual additions I had to make not covered by the Angular migrations. See this [issue](https://github.com/angular/angular-cli/issues/26304#issuecomment-1909967243) for details.